### DISCUSSION TITLE
Using own grids

### DISCUSSION THREAD

**Discussion by geofuzzie** [pyproj4/pyproj#936] [General]
Hello,
within my old python scripts, I used definitions like
mycrs = pyproj.Proj("+init=EPSG:31467 +units=m +nadgrids=./BWTA2017.gsb") to convert coordinates using specific ntv2 grids. After an updaten, using "+init" should be replaced to newer versions of pyproj (pyproj 3.1.0 and proj 7.2.1).
mycrs = CRS('epsg:31467') works fine, but my specific grid is not used.
Is there any possibility to use grid files in the latest version pf pyproj?
Thanks!

**Comment by snowman2** [comment]
See: 
- https://gis.stackexchange.com/a/352300/144357
- https://github.com/pyproj4/pyproj/issues/761

**Reply by geofuzzie** [reply]
Thanks for the fast hint!

The script runs correctly now. There still remains the "FutureWarning: '+init=<authority>:<code>' syntax is deprecated. '<authority>:<code>' is the preferred initialization method. When making the change, be mindful of axis order changes: https://pyproj4.github.io/pyproj/stable/gotchas.html#axis-order-changes-in-proj-6
  projstring = _prepare_from_string(" ".join((projstring, projkwargs)))"

Maybe there will be a solution for this warning in the future.

Thanks!
